packages/arb-ts/src/index.ts

```diff
@@ -16,17 +16,6 @@
 /* eslint-env node */
 'use strict'
 
-export { RollupCreator__factory } from './lib/abi/factories/RollupCreator__factory'
-export { Inbox__factory } from './lib/abi/factories/Inbox__factory'
```

Comment:
does dis mean I can't import these externally from arb-ts anymore?

Reply:
you can import it as in https://github.com/OffchainLabs/arb-token-bridge/pull/211/commits/2e0c13cc62154265f9551673f8e3ea17b05e973a